README.md

```diff
@@ -156,7 +196,7 @@ Params:
 * timestamp - set to null for current timestamp
 
 ```java
-String url = "https://<<URL>>/api/v1/?param1=first&ab-param2=123";
+String url = "https://<<Target_URL>>/api/v1/?param1=first&param2=123";
```

Comment:
Need to share that if they put queryParams formList, they no need to put the query param in the URL ?